support/azure/azure-kubernetes/create-upgrade-delete/akscapacityerror.md

```diff
@@ -20,9 +20,9 @@ When you try to create or start an AKS cluster, you receive one of the following
 
 > **Code:** AKSCapacityError
 >
-> **Message 1:** Creating a new free tier cluster is unavailable at this time in region \<Region\>. In order to create a new cluster, we recommend using an alternate region, or creating a paid tier cluster. For a list of all the Azure regions, visit <https://aka.ms/aks/regions>.
+> **Message 1:** Creating or starting a free tier cluster is unavailable at this time in region \<Region\>. To create a new cluster, we recommend using an alternate region, or create a paid tier cluster. For a list of all the Azure regions, visit <https://aka.ms/aks/regions>. For more details on this error, visit <https://aka.ms/akscapacityerror>.
 >
-> **Message 2:** Creating a new cluster or start cluster is unavailable at this time in region westeurope. To create a new cluster, we recommend using an alternate region. For a list of all the Azure regions, visit <https://aka.ms/aks/regions>.
+> **Message 2:** Creating a new cluster or starting cluster is unavailable at this time in region \<Region>. To create a new cluster, we recommend using an alternate region. For a list of all the Azure regions, visit <https://aka.ms/aks/regions>. For more details on this error, visit <https://aka.ms/akscapacityerror>.
```

Comment:
... or starting an existing cluster ...

Reply:
Yes, I'll make a note we need to update the error messages, but I believe this is how it is currently.